### DISCUSSION TITLE
Покажите, пожалуйста, простой пример для работы с suggestView

### DISCUSSION THREAD

**Discussion by Bambo4a** [yandex-maps-unofficial/vue-yandex-maps#188] [Q&A (2.x)] [answered]
Не понимаю, как использовать. Может кто сделать рабочий пример и показать, как надо все подключать?

**Comment by daniluk4000** [comment, marked as answer]
В версии 2.х (и Я.Картах 3.х) suggestView нет. Можно использовать [API Геосаджеста](https://yandex.ru/dev/geosuggest/doc/ru/), а интерфейс реализовывать придется самостоятельно.
